terraform/providers/aws/images-tester/tester-environment.tf

```diff
@@ -0,0 +1,47 @@
+terraform {
+  required_version = "0.11.11"
```

Comment:
Huumm .. I think we need update the terraform version in all providers supported.

Reply:
Sorry @lborguetti, I made a mistake. Terraform AWS Provider requires terraform 0.10+, so version 0.11.7 is enough. I've already fixed it in this commit: 414cb93.